src/types.ts

```diff
@@ -169,6 +170,9 @@ export interface ExcalidrawProps {
     appState: AppState,
     canvas: HTMLCanvasElement | null,
   ) => void;
+  renderFooter?: (isMobile: boolean) => JSX.Element;
+  lang?: Language["lng"];
+  onLangChange?: (lang: Language["lng"]) => void;
```

Comment:
I'd make the param be of type `Language` to align with `setLanguage` signature.

Also, we should differentiate naming of parameters/props that are `Language` and `Language["lng"]` types. Right now, both are called `lang`. We should call them `language` when of `Language` type, and `lng` when of `Language["lng"]` type.

Ultimately I'd rename `lng` to `code` and call the props `languageCode`. `lng` is non-descriptive and the the difference from `language` non-existent — dunno why we used `lng` to begin with (maybe to align with the language detector).

Reply:
we can rename the prop to `langCode` or `languageCode` for sure but I still would want to pass just the `langCode` and not the whole Language object just to keep the API simple. 
Also `lang` is descriptive as it gives the impression that language is expected.
Thoughts?

Reply:
> but I still would want to pass just the `langCode` and not the whole Language object just to keep the API simple.

I was commenting on `onLangChange`, but GH displayed everything. Meaning, `onLangChange` should be passed the whole deal.